store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonSchemaReader.java

```diff
@@ -115,6 +116,21 @@ public static Schema readSchema(String path) throws IOException {
     return readSchema(path, false, conf);
   }
 
+  /**
+   * Converting carbon schema to arrow schema in byte[],
+   * this schema can be used for reading arrow vector
+   *
+   * @param path
+   * @return
+   * @throws IOException
+   */
+  public static byte[] getArrowSchemaAsBytes(String path) throws IOException {
+    Schema schema = CarbonSchemaReader.readSchema(path).asOriginOrder();
+    ArrowConverter arrowConverter = new ArrowConverter(schema, 1);
```

Comment:
Why the init size is 1  here?

Reply:
It's an initial size, no need to make it configurable. I checked arrow code. They use 0 as initial value. Let me use the same